askup/templates/askup/user_profile.html

```diff
@@ -2,15 +2,18 @@
 
 {% block body %}
 <div class="container-fluid">
+    <div class="row center">
+        <h3>{{ user_organizations|default:"No organizations assigned to the user" }}</h3>
+    </div>
     <div class="row">
         <div class="col-sm-3">
-            <div class="box center dash-profile">
+            <div class="box center dash-profile rounded-10">
                 <img alt="Person" src="/static/assets/default-avatar.png">
                 <br><br>
-                <h3 class="center">{{ first_name }} {{ last_name }}</h3>
+                <h3 class="center">{{ profile_user.first_name }} {{ profile_user.last_name }}</h3>
```

Comment:
What is a user has no First name AND Last name ?

Reply:
fixed this moment by using the function that returns the full user name representation
[<first name> <last name> (]<username>[)]